.sourcekit-lsp/config.json

```diff
@@ -1,4 +1,5 @@
 {
+  "$schema": "https://raw.githubusercontent.com/swiftlang/sourcekit-lsp/refs/heads/main/config.schema.json",
```

Comment:
mostly unrelated change: Add the schema file so VSCode can show if you're using an invalid JSON field.